projects/ion/src/lib/chip/chip.component.scss

```diff
@@ -115,6 +115,10 @@ button {
   margin-bottom: 4px;
 }
 
+.positionIcon {
+  flex-direction: row-reverse;
```

Comment:
I feel this is a risky move, since this will reverse position for all items inside chip, like badges and dropdown icon.
![image](https://user-images.githubusercontent.com/57760325/194866347-5b1264d5-b0f1-4cf4-932a-3ba5f2fffc46.png)
![image](https://user-images.githubusercontent.com/57760325/194866376-aefa24f5-e7b4-4680-8c12-7b6d84f4bce6.png)
If this is a correct behaviour, nice job! If not, you can add more conditionals to your class be setted only if have icons to show.

Reply:
I believe that can be add 2 ion-icons with ngIfs checking if is left or right

Reply:
its wrong, the icons in imgs need be in right side.